go/nbs/table_set.go

```diff
@@ -228,20 +276,19 @@ func (ts tableSet) Rebase(specs []tableSpec) tableSet {
 	}
 
 	// Open all the new upstream tables concurrently
-	openedTables := make(chunkSources, len(tablesToOpen))
+	merged.upstream = make(chunkSources, len(tablesToOpen))
```

Comment:
it seems like when we compact, we remove compactees from upstream. am wrong, or will they re-appear here even if keepCompactions is true?